apimodels/task_log.go

```diff
@@ -54,149 +45,121 @@ type TaskLog struct {
 	Messages     []LogMessage `json:"m"`
 }
 
-func GetSeverityMapping(s int) string {
+func GetSeverityMapping(s level.Priority) string {
 	switch {
-	case s >= int(level.Error):
+	case s >= level.Error:
 		return LogErrorPrefix
-	case s >= int(level.Warning):
+	case s >= level.Warning:
 		return LogWarnPrefix
-	case s >= int(level.Info):
+	case s >= level.Info:
 		return LogInfoPrefix
-	case s < int(level.Info):
+	case s < level.Info:
 		return LogDebugPrefix
 	default:
 		return LogInfoPrefix
 	}
 }
 
-// GetBuildloggerLogsOptions represents the arguments passed into
-// GetBuildloggerLogs function.
-type GetBuildloggerLogsOptions struct {
-	BaseURL       string `json:"_"`
-	TaskID        string `json:"-"`
-	TestName      string `json:"-"`
-	GroupID       string `json:"-"`
-	Execution     *int   `json:"-"`
-	PrintPriority bool   `json:"-"`
-	Tail          int    `json:"-"`
-	LogType       string `json:"-"`
+func GetPriority(prefix string) level.Priority {
```

Comment:
can you add a function comment here (unless this is mega temporary)

Reply:
It doesn't need to be exported so I just made it private, it will also be temporary.